where-for-dinner/accelerator.yaml

```diff
@@ -231,12 +231,15 @@ accelerator:
     dependsOn: 
       name: enableDefaultDevAccount
   - label: Workload URL
-    description: "The URL where the application is published; required when using Security/AppSSO or TAP Spring Cloud Gateway.  Generally 'http<s>://where-for-dinner.<domain name>.  See README for more details."
-    defaultValue: https://where-for-dinner.<UPDATE ME>
+    description: "The URL where the application is published; required when TAP Spring Cloud Gateway.  Generally 'http<s>://where-for-dinner.<domain name>.  See README for more details."
```

Comment:
Should this say "required when using TAP Spring Cloud Gateway" instead of "required when TAP Spring Cloud Gateway"?

Reply:
Good catch.  Adjusted and committed.  Thx!